ACE/include/makeinclude/platform_android.GNU

```diff
@@ -177,3 +177,14 @@ endif
 
 # Link To Android Logging Library for Log_Msg_Android_Logcat
 LIBS += -llog
+
+# link step to avoid 'command line too long' error on Windows
+ifeq ($(OS), Windows_NT)
+  SHOBJS_FILE = $(VSHDIR)tmpfile
+  CLEANUP_OBJS += $(SHOBJS_FILE)
+  define SHLIBBUILD
+    echo " " > $(SHOBJS_FILE)
+    $(foreach myobj, $(VSHOBJS), echo $(myobj) >> $(SHOBJS_FILE);)
```

Comment:
Would `$^` work in place of `$(VSHOBJS)`?  The difference is that other places in the makefiles could add prerequisites to this rule (not sure that ever happens in practice).